src/napari_manual_transforms/_model.py

```diff
@@ -165,6 +179,26 @@ def compact_axis_angle(self):
     def compact_axis_angle(self, ca):
         self.quaternion = rot.quaternion_from_compact_axis_angle(ca)
 
+    @property
+    def translation(self) -> np.ndarray:
+        return self._t
+
+    @translation.setter
+    def translation(self, t: Sequence[float] | np.ndarray):
+        """x,y,z"""
+        self._t = np.asarray(t)
+        self.valueChanged.emit()
+
+    @property
+    def scale(self) -> float:
+        return self._s
+
+    @scale.setter
+    def scale(self, s: float):
+        """scale"""
+        self._s = s
```

Comment:
similar, here. need some simple validation

Reply:
I have implemented a simple user warning with automatically adjusting the value to a minimum value.
Would you rather raise an exception here instead?